src/components/core/Page/header.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import styled from "@emotion/styled";
-
+import PropTypes from 'prop-types';
```

Comment:
We have a convention, that we don't use the default proptypes imports. We try to keep it in our other projects. 

```suggestion
import { string } from 'prop-types';
```